spec/models/post_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.new(name: 'Bianca', photo: 'photo.com.uy', bio: 'developer from Uruguay')
+  user.save
+  post = Post.new(author: user, title: 'First Post', text: 'Welcome to my first post')
+  comment = Comment.new(text: 'Beatifull Post', post:, user:)
+  comment.save
+  like = Like.new(post:, user:)
+  like.save
+
```

Comment:
- [ ] In accordance to the project requirement, kindly implement test for the post's model's method highlighted from your codebase. 👇 

```
 def recents(limit = 5)
    comments.order(created_at: :desc).limit(limit)
  end
```

<img width="419" alt="Untitled" src="https://github.com/BiancaMe/my-sheeps-blog/assets/103386183/e7a02342-d36a-4389-8c74-84a89a7cad1a">